mypy_boto3_builder/stubs_static/types-aioboto3/resources/base.pyi

```diff
@@ -12,8 +12,8 @@ class AIOBoto3ServiceResource(ServiceResource):
     async def __aenter__(self: _R) -> _R: ...
     async def __aexit__(
         self,
-        exc_type: Optional[Type[Exception]],
-        exc_value: Optional[Exception],
+        exc_type: Optional[Type[BaseException]],
+        exc_value: Optional[BaseException],
```

Comment:
Should be moved to https://github.com/youtype/types-aioboto3/blob/main/aioboto3-stubs/resources/base.pyi